### ISSUE TITLE
Npm publish

### ISSUE THREAD

**Issue by mikemellor11** [domharrington/node-gitlog#24]
Hey Dom

I've pulled and closed the pull requests now, could you do an npm publish when you get a second?

Regards

Mike

**Comment by domharrington** [comment]
Just run the tests locally and i appear to be getting an error:

```
  1) gitlog returns R100 & D status for files that are renamed (100 is % of similarity) or A:

      Uncaught AssertionError: expected 'D' to equal 'R100'
      + expected - actual

      -D
      +R100

      at Object.Assertion.equal (node_modules/should/lib/should.js:303:10)
      at test/gitlog.test.js:224:37
      at index.js:9:4622
      at ChildProcess.exithandler (child_process.js:193:7)
      at maybeClose (internal/child_process.js:827:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
```

The `commits[4]` variable in the test looks like:
```
{ status: [ 'D', 'A', 'D', 'A' ],
  files: [ '1-file', '1-fileRename', '2-file', '2-fileRename' ],
  abbrevHash: 'bcb5fe7',
  hash: 'bcb5fe7da63a0a5fc28ce0708b9a084bd0e9b224',
  subject: ' files renamed',
  authorName: 'Your Name' }
```

Do you think we can just remove the tests to do with the status? It's basically just testing git functionality under the hood anyway right? It's set to change in the future in other weird ways too?

**Comment by mikemellor11** [comment]
Hmm, yeah maybe removing the tests would be best, i've tried to find the exact version of git that starts reporting renames as 'R's but i can't seem to pinpoint it. I'd have thought your version of git would definitely flag it correctly like mine below 

```
{ status: [ 'R100', 'D', 'R100', 'D' ],
  files: [ '1-fileRename', '1-file', '2-fileRename', '2-file' ],
  abbrevHash: 'a1e2408',
  hash: 'a1e2408fbee88aa922269ec3cffe2a7ece8c382f',
  subject: ' files renamed',
  authorName: 'Your Name' }
```

But either way its getting the correct status's and there in the right order (even though there A's not R's).

I think removing the test is the best option, either that or drop the git version check and simply check if commit[4].status[0] === 'R100' then do the rename check, otherwise check for output matching yours.

**Comment by domharrington** [comment]
Published on npm at version 2.2.0. Thanks for all your hard work!
